packages/trpc/server/routers/viewer/teams/roundRobin/getRoundRobinHostsToReasign.handler.ts

```diff
@@ -47,7 +48,7 @@ async function getTeamHostsFromDB({
     }),
   };
 
-  const [totalCount, hosts] = await Promise.all([
+  const [totalCount, _hosts] = await Promise.all([
     prisma.host.count({ where: queryWhere }),
     prisma.host.findMany({
```

Comment:
There is no host repository yet, so we can move these queries later there after setting it up